File: tools/templates/test/src/main/python/test7/modules.py
import smv

class M2(smv.SmvModule, smv.SmvOutput):
    def requiresDS(self):
        return [ smv.SmvExtModuleLink("integration.test.test7_1.M1") ]

    def run(self, i):
        return i[ smv.SmvExtModuleLink("integration.test.test7_1.M1") ]

File: tools/templates/test/src/main/scala/test8/modules.scala
package integration.test.test8

import org.tresamigos.smv._

object M2 extends SmvModule("") with SmvOutput {
  override def requiresDS = Seq(SmvExtModuleLink("integration.test.test8_1.modules.M1"))

  override def run(i: runParams) = {
    i(SmvExtModuleLink("integration.test.test8_1.modules.M1"))
  }
}
